lib/checkother.cpp

```diff
@@ -344,24 +344,26 @@ void CheckOther::invalidPointerCast()
             if (fromType->type != toType->type && fromType->type >= ValueType::Type::BOOL && toType->type >= ValueType::Type::BOOL && (toType->type != ValueType::Type::CHAR || printInconclusive)) {
                 if (toType->isIntegral() && fromType->isIntegral())
                     continue;
-                std::string toStr = toType->isIntegral() ? "integer *" : toType->str();
+                std::string toStr = toType->str();
                 toStr.erase(toStr.size()-2);
-                std::string fromStr = fromType->isIntegral() ? "integer *" : fromType->str();
+                if (toStr.rfind("signed ", 0) == 0)
```

Comment:
Is it worth pruning away "signed " from the message?

Reply:
spontaneously it sounds good to keep "signed" to be explicit..